src/components/molecules/PrevNext.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
```

Comment:
I can't seem to see the story for this component in the storybook UI. Can you validate ?